modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+
+In the {logging} 5.8 and later versions, the Loki Operator supports managing access to logs through cluster wide policies, or by namespace. Users who are members of any group specified in the `adminGroups` field are considered admins in the authorization process. Admin access to logs is not restricted by namespace.
+
+.Example admin group settings
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  tenants:
+    mode: openshift-logging # <1>
+    openshift:
+      adminGroups: # <2>
+      - cluster-admin
+      - custom-admin-group
+----
+<1> Custom admin groups are only available in this mode.
+<2> Overrides the default groups (`system:cluster-admin`, `cluster-admin`, `dedicated-admin`)
+
+
+
+.Example cluster wide log access setting
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: logging-all-application-logs-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-logging-application-view # <1>
+subjects: # <2>
+- kind: Group
+  name: system:authenticated # <3>
+  apiGroup: rbac.authorization.k8s.io
+----
+<1> Additional `ClusterRoles` are `cluster-logging-infrastructure-view`, and `cluster-logging-audit-view`.
+<2> Specifies the users or groups this object applies to.
+<3> Specifies all authenticated users.
```

Comment:
3) Specifies all authenticated users/groups

Reply:
@kabirbhartiRH - Would `Specifies all authenticated users regardless of group membership` work? Groups aren't something I think are thought of as authenticating the same way as a user.

Reply:
I think this might be a misunderstanding between Kabir and you. You correctly stated that `system:authenticated` is a group containing `all authenticated users` but Kabir (probably) wanted to hint at the possibility that you can list both User and Group resources as subjects.

Is it maybe a good idea to list both a user and a group in the example?

Reply:
Thanks for the clarification :) On second pass, I think removing callout 3 works, as the 2nd callout covers it, given that the `kind:` is `group`. Thoughts?

Reply:
Yes. looks okay to me